make lint fails

I ran make lint on the latest master and saw this:
`node.go:46:14: wrapperFunc: use strings.ReplaceAll method in `strings.Replace(str, "\n", "", -1)` (gocritic)
		message := strings.Replace(str, "\n", "", -1`

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -50,7 +50,7 @@ jobs:
       name: default
     steps:
       - *fast-checkout
-      - run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.30.0
+      - run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.32.2
       - run: make lint
   check-license:
     executor:

--- a/bitcoin/node.go
+++ b/bitcoin/node.go
@@ -43,7 +43,7 @@ func logPipe(ctx context.Context, pipe io.ReadCloser, identifier string) error {
 			return err
 		}
 
-		message := strings.Replace(str, "\n", "", -1)
+		message := strings.ReplaceAll(str, "\n", "")
 		messages := strings.SplitAfterN(message, " ", 2)
 
 		// Trim the timestamp from the log if it exists

